src/cryptography/hazmat/backends/openssl/backend.py

```diff
@@ -1671,6 +1671,31 @@ def load_elliptic_curve_public_numbers(self, numbers):
 
         return _EllipticCurvePublicKey(self, ec_cdata, evp_pkey)
 
+    def elliptic_curve_exchange_algorithm_supported(self):
+        if self._lib.Cryptography_HAS_EC != 1:
+            return False
+
+        if self._lib.Cryptography_HAS_ECDH != 1:
+            return False
+
+        return True
```

Comment:
This can be simplified to `return self._lib.Cryptography_HAS_EC and self._lib.Cryptography_HAS_ECDH`